src/ui/RadarMap.ts

```diff
@@ -135,6 +135,20 @@ class RadarMap extends maplibregl.Map {
   getMarkers(): RadarMarker[] {
     return this._markers;
   }
+
+  fitToMarkers(fitBoundsOptions: maplibregl.FitBoundsOptions = { padding: 50 }, overrideMarkers?: RadarMarker[]) {
+    const bounds = new maplibregl.LngLatBounds();
```

Comment:
nit: move this down to just before the forEach. No need to initialize if we're going to return early.